include/spineml/simulator/inputValue.h

```diff
@@ -4,6 +4,7 @@
 #include <functional>
```

Comment:
This file's name should probably be in snake-case because that's what we've used elsewhere

Reply:
Slightly confusingly GeNN uses lower-camel case for filenames

Reply:
Haha I forgot this was a PR on the GeNN repo :-p